etc/kayobe/environments/aufn-ceph/kolla/inventory/overcloud-services.j2

```diff
@@ -28,6 +28,13 @@ common
 [elasticsearch-curator:children]
 elasticsearch
```

Comment:
I don't see Neutron OVN agent as mentioned in the title

Reply:
This was just meant to go into Antelope, see PR: https://github.com/stackhpc/stackhpc-kayobe-config/pull/736#pullrequestreview-1710039528